### ISSUE TITLE
can add multi selection from gallary

### ISSUE THREAD

**Issue by gehadfatah** [Dhaval2404/ImagePicker#288]

**Comment by PatricioIN** [comment]
It would be a good feature

**Comment by androidsnippetss** [comment]
How can we upload multiple image

**Comment by waqaruet** [comment]
only missing feature. would be great if multi-images pick is added, like the one matisse have

**Comment by CristianMG** [comment]
I have a pull request sended https://github.com/Dhaval2404/ImagePicker/pull/291 It will required a good testing of all library features

**Comment by PatricioIN** [comment]
Thank you very much for your effort. Is there a way to test your changes via gradle?

**Comment by CristianMG** [comment]
No, I'm sorry, you can clone pull request and launch in a mobile phone, if you would test it, please indicate the test cases in pull request. So mantainers will know it the pull request was properly tested.

**Comment by PatricioIN** [comment]
It works fine!

**Comment by CristianMG** [comment]
Please comment pull request to maintainer take into account. Documenting the tests you have done.

**Comment by Tavorc** [comment]
when the multiple photos feature will available for gradle dependency?

**Comment by narendran12** [comment]
Multiple image picker from gallery feature available now ?
